src/main/java/org/cryptomator/frontend/fuse/mount/AwtFrameworkRevealer.java

```diff
@@ -0,0 +1,23 @@
+package org.cryptomator.frontend.fuse.mount;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+import java.util.function.Consumer;
+
+public class AwtFrameworkRevealer implements Consumer<Path> {
```

Comment:
Not sure if we want this to be a public class. I'd rather have the API consumer to always provide their own revealer and move this class to `src/test/java`. Eventually we could eliminate AWT from the dependency list (thinking about JPMS here).

Reply:
I had the same thought, but its better to discuss this than directly remove it.

Actually, I vote for directly remove it, because the dependency to AWT is one of the reasons for the refactoring of the reveal() method. Of course the test classes must either implement than their own reveal method or just skip this part. (i tend to skip it and remove the invocation of reveal)

Reply:
If you moved this to `test`, the test classes could still use it (just as they would use mocks), but still the jar didn't bring in a dependency as it doesn't include test code.

Reply:
then to `test` it is! like you said, later we can still just remove it.

Reply:
resolved in ef53293